src/main/java/core/basesyntax/parse/FruitRetortImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.parse;
+
+import core.basesyntax.model.Fruit;
+import java.util.Map;
+
+public class FruitRetortImpl implements FruitReport {
+
+    @Override
+    public String create(Map<Fruit, Integer> fruitIntegerMap) {
+        StringBuilder report = new StringBuilder();
+        report.append("fruit,quantity").append(System.lineSeparator());
+        for (Map.Entry<Fruit, Integer> entry : fruitIntegerMap.entrySet()) {
+            report.append(entry.getKey().getName()).append(",")
```

Comment:
We can avoid hard-coding .csv separators by using standard parsing libraries (Apache OpenCSV, for example)

Reply:
fixed